gflownet/proxy/base.py

```diff
@@ -2,8 +2,10 @@
 Base class of GFlowNet proxies
 """
 from abc import abstractmethod
```

Comment:
```suggestion
from abc import abstractmethod, ABC
```
you need to inherit Proxy from ABC to make `@abstractmethod` work (throw error if it isn't implementer in the child class)

Reply:
Thanks!